### ISSUE TITLE
Data persistency handled with DB docker

### ISSUE THREAD

**Issue by iOiurson** [anthill/6element#69]

**Comment by DavidBruant** [comment]
The plan is to make a dockerfile with `FROM postgres:9.4` and add necessary tooling to do backups/restore in that container.

There was doubts about how volumes could/should work with a database.
This is how things have been working for now on MyWebIntelligence, without any problem for now
```
docker rm mywi-dev-db; docker run --name mywi-dev-db -p 5500:5432 -e POSTGRES_PASSWORD=password -v $PWD/_storage/etc:/etc/postgresql -v $PWD/_storage/log:/var/log/postgresql -v $PWD/_storage/data:/var/lib/postgresql -v $PWD/_storage/data/data:/var/lib/postgresql/data -d mywi/database
```
https://github.com/MyWebIntelligence/MyWebIntelligence/blob/8ec6ed200525529da71219dec5e09a7fbaa3148b/package.json#L18

**Comment by 4rzael** [comment]
##Fixed in #79.

Backups are done everyday and last 1 week.

If you want to load a backup, edit the file rebuild-db.yml, uncomment the line ```-BACKUP=backups/save.txt.gz``` and change the name of the backup file.

Then use ```docker-compose -f rebuild-db.yml up```
